test/src/api/commands/element/testSetPassword.js

```diff
@@ -1,4 +1,9 @@
+const path = require('path');
 const assert = require('assert');
+const {Key} = require('selenium-webdriver');
+const common = require('../../../../common.js');
+const {settings} = common;
+const NightwatchClient = common.require('index.js');
```

Comment:
Most of the imports added here can be removed.

Reply:
removed [87fac0c](https://github.com/nightwatchjs/nightwatch/pull/4066/commits/87fac0cfa216805670b78f34432d66952252fd9a)